python/graphscope/_version.py

```diff
@@ -25,5 +25,5 @@
 try:
     with open(os.path.join(git_root, "VERSION")) as f:
         __version__ = f.readline().rstrip()
-except:
+except:  # noqa: E722
```

Comment:
Why do we need the broad except...  I think there might a FileNotFoundError.  or OSError. anything else?

Reply:
Yes, Here we want is cover everything exceptions occur and then set the version `latest`.  Change to `OSError `.